lib/scrabble_word.rb

```diff
@@ -0,0 +1,35 @@
+class ScrabbleWord
+  attr_reader :word
+  def initialize word
+    @word = word.downcase
+    @score = 0
+  end
+
+  def score
+    wordArr = word.split('')
+    wordArr.each do |x|
+      case x
+        when 'a', 'e', 'i', 'l', 'n', 'o', 'r', 's', 't', 'u'
```

Comment:
Nice! How could you DRY this up even further so you don't need to use a bunch of `case...when`? Hint: I'm thinking of JSON and an `each` loop.